internal/pkg/service/appsproxy/dataapps/appconfig/appconfig_test.go

```diff
@@ -284,11 +254,7 @@ func TestLoader_LoadConfig(t *testing.T) {
 					}
 				} else {
 					require.NoError(t, err)
-					assert.Equal(t, attempt.expectedConfig.ID, cfg.ID)
-					assert.Equal(t, attempt.expectedConfig.Name, cfg.Name)
-					assert.Equal(t, attempt.expectedConfig.UpstreamAppURL, cfg.UpstreamAppURL)
-					assert.Equal(t, attempt.expectedConfig.AuthProviders, cfg.AuthProviders)
-					assert.Equal(t, attempt.expectedConfig.AuthRules, cfg.AuthRules)
+					assert.NotEmpty(t, cfg)
 				}
```

Comment:
There is no reason to add new fields, with the same static fields to all test cases.